Source/ThirdParty/ANGLE/extensions/ANGLE_variable_rasterization_rate_metal.txt

```diff
@@ -0,0 +1,62 @@
+Name
+
+    ANGLE_variable_rasterization_rate_metal
+
+Name Strings
+
+    GL_ANGLE_variable_rasterization_rate_metal
+
+Contributors
+
+    Dan Glastonbury, Apple Inc.
+
+Contact
+
+    Dan Glastonbury, Apple Inc. (djg 'at' apple 'dot' com)
+
+Status
+
+    Incomplete
+
+Version
+
+    Last Modified Date: March 30, 2023
+    Author Revision: 1
+
+Number
+
+    OpenGL ES Extension XX
+
+Dependencies
+
+Overview
+
+IP Status
+
+    No known IP claims.
+
+New Types
+
+    /*
+     * GLmtlRasterizationRateMapANGLE is an opaque handle to an id<MTLRasterizationRateMap>
+     */
+    typedef void* GLmtlRasterizationRateMapANGLE;
```

Comment:
This capitalization seems a bit odd - would GLMTLRasterizationRateMapANGLE work?

Reply:
I chose to follow the convention used by `GLeglClientBufferEXT`, `GLeglImageOES` for
`GLmtlRasterizationRateMapANGLE`. I'm not particularly set on the name but was trying to be consistent.